### PR TITLE
V1.1

### PR DESCRIPTION
**Pull request by RoeReRe**
- Updated AboutUs page to include my details
- Updated ReadMe: Mock UI uploaded; most of the other content under this header in the CS2103T webstie should be done (please double check)
- **I did not do anything under the Update Site-Wide settings header (don't think its relevant to MarkBind?)**
- Only updated introductory portion of UG
- Added skeletal PPP for myself (I think can standardise to mine?)
- Updated the DG: Target user profile, value proposition, and user stories

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -4,9 +4,9 @@
   pageNav: 3
 ---
 
-# AB-3 User Guide
+# BayMeds User Guide
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a  Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+BayMeds v.2103 is a **desktop application for managing your medications**. While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface). If you can type fast, BayMeds can get your contact management tasks done faster than traditional GUI apps.
```
Comment (TeeRenJing) [inline docs/UserGuide.md:9]: Would it be better to state what GUI means for non-technical users?

File: README.md [hunk 2]
```diff
@@ -1,14 +1,10 @@
...
   Example usages:
-  * as a starting point of a course project (as opposed to writing everything from scratch)
-  * as a case study
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
-  * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
+  * Track medication needs, dosage and related health information
```
Comment (TeeRenJing) [inline README.md:7]: Should we add a period to the end of these bullet points?

**Review by TeeRenJing — COMMENTED** [review]
LGTM, just a few nits.

File: docs/index.md [hunk 3]
```diff
@@ -3,17 +3,17 @@
...
 
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
-[![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/se-edu/addressbook-level3)
+[![CI Status](https://github.com/AY2324S1-CS2103T-T15-2/tp/actions/workflows/gradle.yml/badge.svg)](https://github.com/AY2324S1-CS2103T-T15-2/tp/actions)
+[![codecov](https://codecov.io/gh/AY2324S1-CS2103T-T15-2/tp/branch/master/graph/badge.svg)](https://codecov.io/gh/AY2324S1-CS2103T-T15-2/tp)
 
 ![Ui](images/Ui.png)
 
-**AddressBook is a desktop application for managing your contact details.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+**BayMeds v.2103 is a desktop application for managing your medications.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
```
Comment (TeeRenJing) [inline docs/index.md:13]: Would v.2103 be confusing for readers? Should we stick to a single name (BayMeds)?

File: README.md (hunk 2 above)
Comment (RoeReRe) [reply README.md:7]: this one i actually not too sure wdyt? Cause the original AB3 UG omitted periods.

Comment (TeeRenJing) [reply README.md:7]: Indeed but ours sounds more like a full sentence. I think if we standardise all periods we also do not have to defend ourselves when others claim it is a bug.

Comment (RoeReRe) [reply README.md:7]: can

**Review by TeeRenJing — APPROVED** [review]
LGTM

**Comment by shyanyong** [comment]
LGTM

**Review by shyanyong — APPROVED** [review]
lgtm

**Pull request merged**